source/developers/concepts/hooks_extension_framework.rst

```diff
@@ -0,0 +1,155 @@
+
+Contained Solution Implementation
+---------------------------------
+
+By avoiding core modifications, the framework promotes self-contained solutions, eliminating the need for custom code to coexist with core logic which lowers maintenance costs for extension developers.
+
+Leveraging the Open edX Plugin Extension Mechanism
+--------------------------------------------------
+
+The framework allows developers to implement custom business logic and integrations directly in plugins. This keeps core modifications minimal, focusing maintenance and development efforts on plugins, where solutions can be built and maintained independently of the core platform.
+
+Standardization
+---------------
+
+Both filters and events implementations implement an approach for adding additional features, such as communication between services or backend flow control. With these standards in place, it’s easy to identify when and how to use the framework as a solution, ensuring a consistent and predictable approach to extending the platform.
+
+Reduce Fork Modifications
+-------------------------
+
+The need to modify logic in forks is minimized, as most extensions can now be implementing using the framework, keeping forks closer to the core and easier to manage.
+
+Community Compatibility
+------------------------
+
+The framework allows for shorter and more agile contribution cycles. By adding standardized extension points, contributors avoid creating customer-specific logic, making development more community-friendly.
+
+Backward Compatibility
+-----------------------
+
+Hooks are designed to be backward compatible, guaranteeing stability across releases and making it easier to upgrade without breaking existing functionality.
+
+Open edX Events and Filters
+============================
+
+Open edX Events
+----------------
+
+Events are Open edX-specific Django signals sent in specific places on the Open edX platform. They allow developers to listen to these signals and perform additional processing based on the event data.
+
+To start using Open edX Events in your project, see the `Open edX Events`_ documentation.
+
+.. _Open edX Events: https://docs.openedx.org/projects/openedx-events/en/latest/
+
+Open edX Filters
+-----------------
+
+Filters are functions that can modify the application's behavior by altering input data or halting execution based on specific conditions. They allow developers to implement application flow control based on their business logic or requirements without directly modifying the application code.
+
+To start using Open edX Filters in your project, see the `Open edX Filters`_ documentation.
+
+.. _Open edX Filters: https://docs.openedx.org/projects/openedx-filters/en/latest/
+
+Differences between Events and Filters
+---------------------------------------
```

Comment:
```suggestion
=============================
```

Reply:
Thank you, done: https://github.com/openedx/docs.openedx.org/commit/a88af4d9f0b50091f95ef1ac2a07c7d1d7bd1872